tools/pyboard.py

```diff
@@ -73,6 +73,7 @@
 import ast
 
 try:
+    separator = os.sep
```

Comment:
Don't need to save this, just use `os.sep` directly later. (I have a mild preference for using `os.path.sep` though as it's clearer what it is)

Reply:
Um, I think you are right, this is not a function value that needs to be calculated, just read it when we use it.